example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+        headers:
+            lightstep-access-token: "${LIGHTSTEP_ACCESS_TOKEN}"
+        # While we expect latency under one second, typically, we
+        # recommend a longer timeout than the default.
+        timeout: 5s
+        # Queue settings are required.  It does not make sense to use
+        # the exporter without a queue, it has to do with
+        # requiring the "num_consumers" limit configured in this
+        # section (i.e., we require a queue in order to limit the
+        # number of concurrent exports).
+        #
+        # Note that the queue settings are applied in unit-terms
+        # produced by the batch processor, so a number like 100 means
+        # the queue has support for 100 pre-batched items.  With up to
+        # 1500 spans each (from the batch processor), this
+        # configuration allows 150,000 spans to occupy memory.
+        sending_queue:
+            enabled: true
+            num_consumers: 4
+            queue_size: 100
+        # Retry settings are optional.
+        #
+        # Note that while retries are attempted, this component will
+        # begin to drop arriving data if the queue is not large
+        # enough.
+        retry_on_failure:
+            # We recommend disabling retries, since while the export is
+            # blocked it is likely that arriving spans will drop, and
+            # Otherwise, collectors will need substantial additional
+            # memory to survive transient failures.  Nevertheless, we
+            # recommend a limited retry policy to gracefully occasional
+            # failures, paired with a modest queue size.
+            #
+            # Note there is a persistent storage option inherited from a
+            # common collector component.  When persistent storage is
+            # configured, the default retry configuration is sensible.
+            #
+            # For more details on retry and queue settings, please refer to
+            # https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md
+            enabled: true
+            max_elapsed_time: 30s
+    debug:
+        verbosity: basic
+        sampling_initial: 5
+        sampling_thereafter: 200
+
+extensions:
+    # The healthcheck extension provides an endpoint that provides
+    # support to run liveness checks for the Collector.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md
+    health_check:
+    # If this collector is not running in Kubernetes, we recommend adding
+    # this opampextension to report its config and status.
+    # For more details, see https://docs.lightstep.com/docs/monitor-collector-health
+    opamp:
+        server:
+            ws:
+                endpoint: "wss://opamp.lightstep.com/v1/opamp"
+                headers:
+                    "Authorization": "bearer ${LS_OPAMP_API_KEY}"
+
+service:
```

Comment:
Description for service:

> The service section configures the enabled components in the Collector based on the configuration found in the receivers, processors, exporters, and extensions sections.

This config defines the trace, metric, and log pipelines, the Cloud Observability default extensions, and Collector telemetry. If a component is configured but not defined in the service section, then it's not enabled.